flow_merge/lib/merge_settings.py

```diff
@@ -70,13 +72,24 @@ class HfHubSettings(BaseModel):
 
     @field_validator("token")
     def validate_token(cls, v):
-        if v is False:  # FIXME: exec here the regexp checker for the token format
-            logger.warning(
-                f"Probably invalid Hugging Face Hub token. HF token is usually of the form '<REGEXP_HERE>'."
-            )
+        if v:
+            # Check the token format using a regular expression
+            token_pattern = r"^hf_(?=.*[a-z])(?=.*[A-Z])[a-zA-Z]+$"
+            if not re.match(token_pattern, v):
+                logger.warning(
+                    f"Invalid Hugging Face Hub token format. HF token should be of the form '{token_pattern}'."
+                )
+                return v
+
+            # Attempt to login with the provided token
+            try:
+                login(token=v)
```

Comment:
To remember in the testing implementation that we have a networked call in the field validation and to get around it.

Reply:
> To remember in the testing implementation that we have a networked call in the field validation and to get around it.

I think we should remove it from config and validations and use HF_TOKEN only, as you suggested in #15